motionit/src/main/java/com/back/motionit/domain/challenge/room/dto/CreateRoomResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.back.motionit.domain.challenge.room.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import com.back.motionit.domain.challenge.room.entity.ChallengeRoom;
+import com.back.motionit.domain.challenge.video.entity.ChallengeVideo;
+import com.back.motionit.domain.challenge.video.entity.OpenStatus;
+import com.back.motionit.domain.user.entity.User;
+
+public record CreateRoomResponse(
+	Long id,
+	User user,
```

Comment:
```suggestion
	 Long userId, 
```
User 엔티티의 민감 정보가 노출될 위험이 있을거 같습니다.